opt/resource/out

```diff
@@ -67,22 +67,30 @@ def get_mime_subtype(payload):
 
 
 def get_subject(src, payload, template_vars):
-    fname = get_value_from_payload(payload, 'params', 'subject')
-    with open(os.path.join(src, fname)) as fp:
-        return Template(fp.read()).render(template_vars)
+    subject_text = get_value_from_payload(payload, 'params', 'subject_text', optional=True)
+    if not subject_text:
+        fname = get_value_from_payload(payload, 'params', 'subject')
+        with open(os.path.join(src, fname)) as fp:
+            return Template(fp.read()).render(template_vars)
```

Comment:
I would rather use a variable `subject` for both config values `'subject_text'` and `'subject'` and assign its value accordingly. Additionally I think that the template variables should be applied in any case.

```python
    subject = get_value_from_payload(payload, 'params', 'subject_text', optional=True)
    if not subject:
        fname = get_value_from_payload(payload, 'params', 'subject')
        with open(os.path.join(src, fname)) as fp:
            subject = fp.read()
    return Template(subject).render(template_vars)
```